src/custom_types.rs

```diff
@@ -0,0 +1,77 @@
+use lsp_data::Range;
```

Comment:
If I remember correctly it's the serialization derives. One is `RustcSerialize` and the other is `Serializable`. I'll double check after work.